scrapers_next/ny/people.py

```diff
@@ -158,3 +158,95 @@ def process_item(self, item):
         p.capitol_office.voice = capitol_addr["phone"] or ""
         p.capitol_office.fax = capitol_addr["fax"] or ""
         return p
+
+
+class Senate(HtmlListPage):
+    """
+    Contact information is harder to collect in a reasonable manner
+    """
+
+    source = URL("https://www.nysenate.gov/senators-committees")
+    selector = CSS("div.c-senator-block", num_items=63)
+
+    def _parties(self, party):
+        if party == "(D)" or party == "(D, IP)":
+            return "Democratic"
+        if party == "(R)":
+            return "Republican"
+        if party == "(R, C, IP, RFM)":
+            return "Republican/Conservative/Independence/Reform"
+        if party == "(D, WF)":
+            return "Democratic/Working Families"
+        if party == "(R, C, IP, LIBT)":
+            return "Republican/Conservative/Independence/Libertarian"
+        if party == "(D, IP, WF)":
+            return "Democratic/Independence/Working Families"
+        if party == "(R, C)":
+            return "Republican/Conservative"
+        if party == "(R, C, IP)":
+            return "Republican/Conservative/Independence"
+        # if party == "(D, IP)":
+        #     return "Democratic/Independence"
+        return party
+
+    def process_item(self, item):
+        """
+                      <div class="u-even">
+              <a href="/senators/fred-akshar">
+                <div class="c-senator-block">
+                        <div class="nys-senator--thumb">
+                                <img src="https://www.nysenate.gov/sites/default/files/styles/160x160/public/01-10-20_100-01_0011_edited_0.jpg?itok=k-SCUDnr" width="160" height="160" alt="" />		</div>
+                        <div class="nys-senator--info">
+                                <h4 class="nys-senator--name">Fred Akshar</h4>
+                                <span class="nys-senator--district">
+                                        <span class="nys-senator--party">
+                                        (R, C, IP, RFM)				</span>
+                                                                                52nd District							</span>
+                        </div>
+                </div>
+        </a>
+
+            </div>
+        """
+        img = CSS("div.nys-senator--thumb img").match_one(item).get("src")
+        name = (
+            CSS("div.nys-senator--info h4.nys-senator--name")
+            .match_one(item)
+            .text_content()
+        )
+        party = (
+            CSS(
+                "div.nys-senator--info span.nys-senator--district span.nys-senator--party"
+            )
+            .match_one(item)
+            .text_content()
+            .strip()
+        )
+        district = (
```

Comment:
District needs to just be the number value, but the rest looks good!

Reply:
So our example should be `52` and not `52nd District`?

Reply:
Yup